Java/ds/SparseMatrices.java

```diff
@@ -0,0 +1,120 @@
+//          SPARSE MATRIX
+
+
+//  Java program for Sparse Matrix Representation using Array 
+import java.util.*;
+
+public class SparseMatrices{
+
+    public static void main(String[] args) {
```

Comment:
Use main() function only for the driver code. Use a separate function for putting all your logic

Reply:
Changed as per the requirments